src/lettuce/plugins/welcome.py

```diff
@@ -0,0 +1,48 @@
+from machine.plugins.base import MachineBasePlugin
+from machine.plugins.decorators import process
+
+
+class WelcomePlugin(MachineBasePlugin):
+    @process("member_joined_channel")
+    async def welcome(self, event):
+        user_id = event['user']
+        channel_id = event['channel']
```

Comment:
Is it intended to send the welcome message for any Slack channel?

Reply:
just will send the welcome massage on those channel where this app is available

Reply:
> I think it should send the message whenever anyone joins the Slack server currently we detect this and send a message to Slack. You can see the existing logic and maybe use that to determine when to send the message use.

@DonnieBLT sir, should we send this welcome text to the User DM when they join ?

 and is there anything we should do for when any user join the channel where this bot is available ?